### DISCUSSION TITLE
There is no extension able to load the configuration for "when@prod"

### DISCUSSION THREAD

**Discussion by Fabianf1983** [kimai/kimai#4406] [On-Premise: Installation & Updates]
### Describe the issue

I dont know what todo. Wasted already 5hrs to find the problem.

$ sudo php bin/console kimai:update

In YamlFileLoader.php line 722:

  There is no extension able to load the configuration for "when@prod" (in "C
  :\xampp8\htdocs\Kimai\kimai2/config/packages\monolog.yaml"). Looked for nam
  espace "when@prod", found ""framework", "doctrine", "doctrine_migrations",
  "sensio_framework_extra", "monolog", "twig", "security", "admin_lte", "jms_
  serializer", "fos_rest", "nelmio_api_doc", "nelmio_cors", "webpack_encore",
   "twig_extra", "handcraftedinthealps_rest_routing", "kimai"".

PHP 8.2.1
Upgrade from 1.20.4 

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

1.20.4

### How do you run Kimai?

Shared-Hosting

### Which PHP version are you using?

8.2

**Comment by kevinpapst** [comment]
Thanks for trying to fix the issue by yourself before posting! Sad you missed this part in the docs:
<img width="1183" alt="Bildschirmfoto 2023-11-05 um 12 47 33" src="https://github.com/kimai/kimai/assets/533162/2472ff8e-ba43-4959-8447-c1759357ffd1">

Try
```
rm -r var/cache/*
```
as mentioned in https://www.kimai.org/documentation/updates.html

**Comment by Fabianf1983** [comment]
Thats what I did

![image](https://github.com/kimai/kimai/assets/143863941/c052100a-c420-4528-bcb4-85ae94291bb9)

git checkout 2.2.1
![image](https://github.com/kimai/kimai/assets/143863941/d48f9420-f5c4-4405-ae22-6c3862968efb)

also this line will give me a warning
composer install --optimize-autoloader -n
![image](https://github.com/kimai/kimai/assets/143863941/6f4bd64b-1162-46a8-9ea4-c7b01ab70ebb)

the update will end with
![image](https://github.com/kimai/kimai/assets/143863941/6fa16cdc-9033-4ce0-bfc2-69fa2f638e75)


So I did excatly what the documentation says.

**Reply by kevinpapst** [reply]
You bash clearly says that you are on 1.20.4 but the second screenshot shows that you have some git issue, that needs to be fixed:
https://www.kimai.org/documentation/updates.html#changed-files

**Reply by Fabianf1983** [reply]
I installed new, linked the database and upgraded, still wont run fine. KimaiWebloader wont load.

**Reply by kevinpapst** [reply]
You are disappointed? I am so sorry, really, that I spent my lifetime to give you a free product and support. 💩

**Reply by Fabianf1983** [reply]
I dont say the product is crap. I really like it coz its useable (when it run) and I really appreciate your work and support.
It just cost me again one day of my free time.
